src/components/pages/link-external-account.yml

```diff
@@ -0,0 +1,46 @@
+welcomeBack:
+  en: Welcome (back)!
+  de: Willkommen (zurück)!
+choicesDescription:
+  en: |
+    Please connect your Shibboleth login with your existing {appName} account or
+    create a new account in case you have logged in for the first time.
```

Comment:
How about '...in case this is your first log in' ?

Reply:
Sure, changed